fyle_accounting_mappings/models.py

```diff
@@ -650,3 +650,37 @@ def create_or_update_category_mapping(
         )
 
         return category_mapping
+
+
+class ExpenseFields(models.Model):
```

Comment:
don't we need parent field id?

Reply:
parent field id will be stored in mapping setting table forget that will add that too